tests/app/celery/test_contact_information_tasks.py

```diff
@@ -78,9 +74,19 @@ def test_should_get_phone_number_and_update_notification(client, mocker, notific
     assert notification.to == '+15555555555'
 
 
-def test_should_not_retry_on_non_retryable_exception(client, mocker, notification):
+def test_should_not_retry_on_non_retryable_exception(client, mocker, sample_template, sample_notification):
```

Comment:
"" test naming

Reply:
We are changing our approach to testing.